thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/detection/yaml/YamlResource.java

```diff
@@ -0,0 +1,122 @@
+package com.linkedin.thirdeye.detection.yaml;
+
+import com.google.common.base.Preconditions;
+import com.google.common.base.Strings;
+import com.linkedin.thirdeye.datalayer.bao.DetectionAlertConfigManager;
+import com.linkedin.thirdeye.datalayer.bao.DetectionConfigManager;
+import com.linkedin.thirdeye.datalayer.dto.DetectionAlertConfigDTO;
+import com.linkedin.thirdeye.datalayer.dto.DetectionConfigDTO;
+import com.linkedin.thirdeye.datalayer.util.Predicate;
+import com.linkedin.thirdeye.datasource.DAORegistry;
+import com.linkedin.thirdeye.detection.ConfigUtils;
+import com.wordnik.swagger.annotations.ApiParam;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.core.Response;
+import org.apache.commons.collections.MapUtils;
+import org.yaml.snakeyaml.Yaml;
+
+
+@Path("/yaml")
```

Comment:
I wonder if we should merge this with the "migration" resource somehow. Also turn *something* into a detection config. Thoughts?

Reply:
Maybe keep separate so that we can remove the migration resource after fully migrated to the new pipeline?